### PR TITLE
Entities local layers

### PR DESCRIPTION
**Pull request by maxlath**
Client side PR for https://github.com/inventaire/inventaire/pull/753

### CODE REVIEW

File: app/modules/entities/components/layouts/entity_layout_actions.svelte [hunk 1]
```diff
@@ -55,15 +54,8 @@
       <span class="button-text">{I18n('refresh Wikidata data')}</span>
     </button>
   </li>
-  <li>
-    <Link
-      url={wikidataHistoryUrl}
-      text={I18n('entity history')}
```
Comment (jum-s) [inline app/modules/entities/components/layouts/entity_layout_actions.svelte:61]: Looks like there is still a need to have a link to Wikidata history page for non-local layer entities, ie `5/entity/wd:Q42511/history`

File: app/modules/entities/components/layouts/entity_layout_actions.svelte [hunk 2]
```diff
@@ -12,16 +12,15 @@
 
   let waitForEntityRefresh
 
-  const { uri, type, claims } = entity
+  const { uri, wdUri, type, claims } = entity
```
Comment (jum-s) [inline app/modules/entities/components/layouts/entity_layout_actions.svelte:15]: After building the server types on `same-as` branch, there is still some TS errors:

```
/client/app/modules/entities/components/layouts/entity_layout_actions.svelte:15:16
Error: Property 'wdUri' does not exist on type 'SerializedEntity'. (ts)

  const { uri, wdUri, type, claims } = entity



/client/app/modules/entities/components/editor/entity_edit_menu.svelte:18:34
Error: Property 'invUri' does not exist on type 'SerializedEntity'. (ts)
  let flash
  const { uri, isWikidataEntity, invUri, wdUri, label, historyPathname } = entity
  const { hasDataadminAccess } = app.user


/client/app/modules/entities/components/editor/entity_edit_menu.svelte:18:42
Error: Property 'wdUri' does not exist on type 'SerializedEntity'. (ts)
  let flash
  const { uri, isWikidataEntity, invUri, wdUri, label, historyPathname } = entity
  const { hasDataadminAccess } = app.user


/client/app/modules/entities/components/wikidata_edit_intro.svelte:18:19
Error: Property 'id' does not exist on type 'SerializedWdEntity'. (ts)
    wikidataOauth = API.auth.oauth.wikidata + `&redirect=${editPathname}`
    wdId = entity.id
    wikiUrl = getWdWikiUrl(wdId)
```

Comment (maxlath) [reply app/modules/entities/components/layouts/entity_layout_actions.svelte]: Fixed in 85b15c87d

**Review by jum-s — APPROVED** [review]

File: app/modules/entities/components/layouts/entity_layout_actions.svelte (hunk 1 above)
Comment (maxlath) [reply app/modules/entities/components/layouts/entity_layout_actions.svelte:61]: Done in 7c67b070f

**Pull request merged**